src/dispatch/workflow/enums.py

```diff
@@ -0,0 +1,9 @@
+from dispatch.enums import DispatchEnum
+
+
+class WorkflowInstanceStatus(DispatchEnum):
```

Comment:
@kevgliss should the values be in capital letters?

Reply:
They could be... I believe they get rendered as "Workflow foo is {status}".

Reply:
Ok. I'll convert them to capital, so we're consistent with other enums and messaging.